scripts/Modules/mkw_core.py

```diff
@@ -88,3 +88,53 @@ def get_speed(playerIdx=0):
     xyz = math.sqrt(x ** 2 + y ** 2 + z ** 2)
 
     return speed(x, y, z, xz, xyz)
+
+# Next 3 functions are used for exact finish display
+def getIGT(lap, player):
+    totalTime = [0, 0]
+    totalTime[0] = chase_pointer(
+        classes.getRaceInfoHolder(), [0xC, (player)*0x4, 0x3C, (lap-1)*0xC + 0x5], 'u8')
+    totalTime[1] = chase_pointer(
+        classes.getRaceInfoHolder(), [0xC, (player)*0x4, 0x3C, (lap-1)*0xC + 0x6], 'u8')
```

Comment:
I would like for this function to get simplified for a bit just due to how similar the two chase_pointer calls are. For one, you could just do a single 'u16' read instead of two 'u8' reads, and just split the upper and lower 8 bits into the separate values.